.github/workflows/deploy-docs-workflow.yml

```diff
@@ -0,0 +1,12 @@
+name: Deploy docs
+description: Dispatch Netlify production deployment
+
+on:
+  workflow_call:
+
+jobs:
+  deploy:
+    runs-on: ubicloud-standard-2
+    steps:
+      - name: Dispatch docs deployment
+        run: curl -X POST -d {} https://api.netlify.com/build_hooks/60e71936b15bb431b42b698a
```

Comment:
Nit - a link to the [netlify documentation that specifies this command](https://docs.netlify.com/configure-builds/build-hooks/) might help to avoid someone wondering what it does.